### PR TITLE
Add a proposed November London Tech Day

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: _events/sr2020/london-tech-day-november.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+type: techday
+location: Thread, Whitechapel
+---
+
+Tech Days are opportunities for teams to get help if they're struggling with a
+technical issue, and is a controlled environment for students to make more
+progress on their robots. It's also an opportunity to see how well they're doing
+in comparison to other teams.
+
+London Tech Days will only run of there is sufficient interest. Spaces are
```
Comment (kierdavis) [inline _events/sr2020/london-tech-day-november.md]:
```suggestion
London Tech Days will only run if there is sufficient interest. Spaces are
```

File: _events/sr2020/london-tech-day-november.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: London November Tech Day
+date: 2019-11-30 10:00:00
+layout: event
+type: techday
+location: Thread, Whitechapel
+---
+
+Tech Days are opportunities for teams to get help if they're struggling with a
+technical issue, and is a controlled environment for students to make more
```
Comment (kierdavis) [inline _events/sr2020/london-tech-day-november.md]:
```suggestion
technical issue, and are a controlled environment for students to make more
```

File: _events/sr2020/london-tech-day-november.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Upon arrival at reception, please ask for Thread.
+
+## Schedule
+
+| Time  | Info |
+|-------|------|
+| 10:00 | Doors Open. |
+| 12:30 | There will be a lunch break. Teams should bring their own lunches or get them from food vendors near the venues. |
+| 13:00 | Lightning Talks, where teams will talk about their robots and what they plan on doing with them. |
```
Comment (kierdavis) [inline _events/sr2020/london-tech-day-november.md]:
```suggestion
| 13:00 | Lightning Talks, where teams will briefly talk about their robots and what they plan on doing with them. |
```

Let's not make this sound too scary.

File: _events/sr2020/london-tech-day-november.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+1 Alie Street,  
+Whitechapel,  
+London,  
+E1 8DE
+
+The easiest way to travel will be by public transport -- Aldgate, Aldgate East
+and Tower Gateway stations are all within a five minute walk, and Liverpool
+Street station is a ten minute walk away. There are also a large number of
+busses which stop nearby.
```
Comment (trickeydan) [inline _events/sr2020/london-tech-day-november.md]:
```suggestion
buses which stop nearby.
```

https://www.grammarly.com/blog/busses-buses/

**Comment by PeterJCLaw** [comment]
Woah, that's a load of grammar things we missed last year.

File: _events/sr2020/london-tech-day-november.md (hunk 1 above)
Comment (PeterJCLaw) [reply _events/sr2020/london-tech-day-november.md]: cb11b30

File: _events/sr2020/london-tech-day-november.md (hunk 2 above)
Comment (PeterJCLaw) [reply _events/sr2020/london-tech-day-november.md]: cb11b30

File: _events/sr2020/london-tech-day-november.md (hunk 3 above)
Comment (PeterJCLaw) [reply _events/sr2020/london-tech-day-november.md]: 2ce62f4 drops this (see commit for why)

File: _events/sr2020/london-tech-day-november.md (hunk 4 above)
Comment (PeterJCLaw) [reply _events/sr2020/london-tech-day-november.md]: "busses" is not wrong so I'm going to leave this.

**Comment by PeterJCLaw** [comment]
I've made a booking for this. I'll merge this tomorrow unless I hear any objections to the booking.

**Pull request merged**